Specs/DateTime/English/DateTimeModel.json

```diff
@@ -4018,4 +4018,47 @@
     }
   }]
 }
+,
+{
+  "Input": "Can I do a booking for the 09th of May for 2 nights?",
+  "Context": {
+    "ReferenceDateTime": "2018-06-22T00:00:00"
+  },
+  "Results": [{
+    "Text": "09th of may",
+    "Start": 27,
+    "End": 37,
+    "TypeName": "datetimeV2.date",
+    "Resolution": {
+      "values": [
+        {
+          "timex": "XXXX-05-09",
+          "type": "date",
+          "value": "2018-05-09"
+        },
+        {
+          "timex": "XXXX-05-09",
+          "type": "date",
+          "value": "2019-05-09"
+        }
+      ]
+    }
+  },
+  {
+    "Text": "nights",

```

Comment:
This should extract "2 nights". Let's open an issue for a better recognition in this case.